modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/StyleTransformFunctionTest.java

```diff
@@ -16,15 +16,15 @@
  */
 package org.geotools.mbstyle.transform;
 
-import java.io.IOException;
-
 import org.geotools.mbstyle.MBStyle;
 import org.geotools.mbstyle.MapboxTestUtils;
 import org.geotools.styling.StyledLayerDescriptor;
 import org.json.simple.JSONObject;
 import org.json.simple.parser.ParseException;
 import org.junit.Test;
 
+import java.io.IOException;
+
```

Comment:
Only explanation is the difference in IDE's used, IntelliJ vs Eclipse.  looks like one of them rearranged imports and since git reported it as being modified, I included it.